source/solvers/cahn_hilliard.cc

```diff
@@ -44,30 +46,35 @@ CahnHilliard<dim>::setup_assemblers()
   this->assemblers.push_back(
     std::make_shared<CahnHilliardAssemblerAngleOfContact<dim>>(
       this->simulation_control,
+      this->simulation_parameters.multiphysics.cahn_hilliard_parameters,
+      (this->simulation_parameters.multiphysics.cahn_hilliard_parameters
```

Comment:
Nice ! :) 
but instead of repeating the if 3 times, wouldn't it be better to initialize a double once before and entering it as an argument.

Reply:
I thought so too but I want to keep the `setup_assemblers` function as "clean" as possible, and not have computations on the side. I don't think the three ifs have a big impact on performance, but I am open to discuss this